### DISCUSSION TITLE
Is the Forge port abandoned? or are you still working on it? :)

### DISCUSSION THREAD

**Discussion by grayWolfGameingNo2** [Ladysnake/Requiem#576] [Q&A] [answered]
Hello Derailedzack, Are you still working on the 1.16.5 port? Me along with a lot of people would love to have this port for their server/moddpack/solo-play! And if you are still working on it... would you like to have some help with it? I don't know much, but I would love to try to help!

**Comment by Redfan2** [comment, marked as answer]
This is the discussion board for the original Requiem, not it‘s ports. Ladysnake is not related to this port
